multiclusterhub/charts/application-chart/templates/applicationconsole-service.yaml

```diff
@@ -0,0 +1,31 @@
+# Licensed Materials - Property of IBM
+# (C) Copyright IBM Corporation 2016, 2019 All Rights Reserved
+# US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+# Copyright Contributors to the Open Cluster Management project
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: {{ template "application.fullname" . }}-applicationui-secrets
+  name: {{ template "application.fullname" . }}-applicationui
+  labels:
+    app: {{ template "application.name" . }}
+    component: "applicationui"
+    chart: {{ template "application.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "application.name" . }}
+    helm.sh/chart: {{ template "application.chart" . }}
+spec:
+  ports:
+    - port: 3001
+      targetPort: 3001
+      protocol: TCP
+      name: http
+  selector:
+    app: {{ template "application.name" . }}
+    component: "applicationui"
+    release: {{ .Release.Name }}
+  
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822